R/model.R

```diff
@@ -304,13 +306,22 @@ CmdStanModel$set("public", name = "compile", value = compile_method)
 #'   _Euclidean Metric_ section of the CmdStan documentation for more details.
 #'   One of the following:
 #'     - A single string from among `"diag_e"`, `"dense_e"`, `"unit_e"`.
-#'     - A character vector containing paths to files (one per chain) compatible
-#'     with CmdStan that contain precomputed metrics. Each path must be to a
-#'     JSON or Rdump file that contains an entry `inv_metric` whose value is
-#'     either the diagonal vector or the full covariance matrix. If
+#'   * `inv_metric`: (vector, matrix) A vector (if metric == 'diag_e') or a
+#'     matrix (if metric == 'dense_e') for initializing the inverse metric. The
+#'     vector is interpreted as a diagonal metric. The inverse metric is
+#'     usually set to an estimate of the posterior covariance. If
 #'     `adapt_engaged=TRUE`, Stan will use the provided metric just as an
-#'     initial guess during adaptation. To turn off adaptation when using a
-#'     precomputed metric set `adapt_engaged=FALSE`.
+#'     initial guess during adaptation. To turn off adaptation when using
+#'     a precomputed metric set `adapt_engaged=FALSE`. Cannot be used in
+#'     conjunction with metric_file.
```

Comment:
Can you add the single back ticks "`" for markdown code formatting around "metric_file"? That way it gets rendered properly in the help pages.

Reply:
I'm doing cntrl+shift+B to rebuild and reinstall cmdstanr, but when I do '?`model-method-sample`' to check the docs, I don't see the updated docs. Is there something else I should be doing? Or am I looking at the wrong docs?

Reply:
Ah, good question. Two options:

1) Run `devtools::document()` before rebuilding with `cntrl+shift+B`

2) If you want it to automatically do it when rebuilding (which I do, but I think has to be set individually): 
  - open the RStudio project options (for me on my mac if I click on the rstudio projects list in the top right corner then "Project Options" is the last thing in the list after all the project names but might be different for you)
  - Once in the project options interface go to the "Build Tools" panel
  - Make sure "Generate documentation with roxygen" has a check mark and then click the "Configure..." button. 
  - In the window opened by "Configure..." make sure everything listed under "Automatically roxygenize when running:" has a check mark